airflow/macros/__init__.py

```diff
@@ -15,7 +15,6 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-"""Macros."""
```

Comment:
![image](https://user-images.githubusercontent.com/34150/128382152-35023b5f-352f-430f-a065-1de39b51553c.png)

This showed up as `Macros` in the docs output, and now we don't have pylint this should be fine :)